vmss/vmss-setup-deploy.yml

```diff
@@ -36,6 +57,13 @@
       src: "tomcat-users.xml"
       dest: "/var/lib/tomcat8/conf/tomcat-users.xml"
 
+  - name: copy app to azure vm
```

Comment:
Start task name with uppercase "Copy application to Azure VM"
And also i think we should start things like "Azure" from uppercase and "VM" in uppercase everywhere